test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -223,17 +234,16 @@ class cursor_bound_01 : public test {
                 }
                 break;
             }
-
+            /* Make sure that records match between both cursors. */
+            testutil_check(normal_cursor->get_key(normal_cursor.get(), &normal_key));
+            testutil_check(range_cursor->get_key(range_cursor.get(), &range_key));
+            testutil_assert(std::string(normal_key).compare(range_key) == 0);
```

Comment:
For consistency throughout this file, should we be using `testutil_assert(std::string(normal_key).compare(range_key)` or `testutil_assert(normal_key.compare(range_key))`? See lines 329 and 420.

Reply:
In this case the `normal_key` is a char* and need to be cast to a std::string.

Reply:
I thought the same @clarissecheah but then realized we could not in this case 😛